examples/namespaced/object/testhooks/disable-ssa.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+set -aeuo pipefail
+
+echo " ⏳ Disabling SSA feature for the provider..."
+${KUBECTL} patch deploymentruntimeconfig runtimeconfig-provider-kubernetes --type='json' -p='[{"op":"replace","path":"/spec/deploymentTemplate/spec/template/spec/containers/0/args", "value":["--debug", "--no-enable-server-side-apply"]}]'
```

Comment:
> --no-enable-server-side-apply

Oh, interesting, can we set this flag to false like this? vs `--enable-server-side-apply=false`